app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1249,19 +1330,27 @@ class MultipleDisposalsTriageController @Inject() (
 
           case Left(startingNewDraftReturn) =>
             answers match {
-              case _: IncompleteMultipleDisposalsTriageAnswers =>
+              case _: IncompleteMultipleDisposalsTriageAnswers      =>
                 Redirect(
                   routes.MultipleDisposalsTriageController.checkYourAnswers()
                 )
 
-              case c: CompleteMultipleDisposalsTriageAnswers   =>
-                val newDraftReturn = DraftMultipleDisposalsReturn
-                  .newDraftReturn(
-                    uuidGenerator.nextId(),
-                    c,
-                    startingNewDraftReturn.representeeAnswers
-                  )
-                val newJourney     = FillingOutReturn(
+              case complete: CompleteMultipleDisposalsTriageAnswers =>
+                val newDraftReturn =
+                  if (complete.assetTypes.contains(IndirectDisposal))
```

Comment:
this should be `complete.assetType === List(IndirectDisposal)` - we only want in indirect journey if they selected indirect disposals only